src/Streamly/Internal/Data/Stream/SVar/Eliminate.hs

```diff
@@ -43,7 +43,7 @@ import Streamly.Internal.Data.Time.Clock (Clock(Monotonic), getTime)
 import qualified Streamly.Internal.Data.Stream.StreamD.Type as D
     (Stream(..), Step(..), fold)
 import qualified Streamly.Internal.Data.Stream.StreamK.Type as K
-    (Stream(..), mkStream, foldStream, foldStreamShared, nilM)
+    (Stream, mkStream, foldStream, foldStreamShared, nilM)
```

Comment:
What is `Stream` here? Why are we using `Stream` anymore?

Reply:
StreamK, we wanted to change the least possible in Svar module.